packages/-ember-data/tests/unit/record-arrays/record-array-test.js

```diff
@@ -97,6 +97,46 @@ module('unit/record-arrays/record-array - DS.RecordArray', function (hooks) {
     assert.strictEqual(recordArray[3], undefined);
   });
 
+  test('#filterBy', async function (assert) {
```

Comment:
you'll want to use `deprecatedTest` vs `assert.expectDeprecation` both so we know to clean it up later and because it will then get skipped when deprecations are removed and we run the tests.

Reply:
this is a perfectly fine place for this test btw

Reply:
🆒 thanks. Fixed.